pkg/sql/catalog/systemschema/system.go

```diff
@@ -1007,6 +1007,145 @@ CREATE TABLE system.span_stats_tenant_boundaries (
 	  FAMILY "primary" (crdb_region, unavailable_at)
 ) ;
 `
+
+	TxnExecutionStatsTableSchema = `
+	CREATE TABLE system.transaction_execution_insights (
+		transaction_id                     UUID NOT NULL,
+		transaction_fingerprint_id         BYTES NOT NULL,
+		query_excerpt                      STRING,
+		implicit_txn                       BOOL,
+		session_id                         STRING NOT NULL,
+		start_time                         TIMESTAMPTZ,
+		end_time                           TIMESTAMPTZ,
+		user_name                          STRING,
+		app_name                           STRING,
+		user_priority                      STRING,
+		retries                            INT8,
+		last_retry_reason                  STRING,
+		problems                           INT[], 
+		causes                             INT[],
+		stmt_execution_ids                 STRING[],
+		cpu_sql_nanos                      INT8,
+		last_error_code                    STRING,
```

Comment:
The error message was also added to the vtables in this commit - https://github.com/cockroachdb/cockroach/pull/110565.
Do we want to add that to the system tables too?